src/current/v25.1/set-up-logical-data-replication.md

```diff
@@ -85,87 +96,148 @@ When you run LDR in [`immediate` mode](#modes), you cannot replicate a table wit
     ~~~
 
 ## Step 3. Start LDR
 
-In this step, you'll start the LDR job from the destination cluster. You can replicate one or multiple tables in a single LDR job. You cannot replicate system tables in LDR, which means that you must manually apply configurations and cluster settings, such as [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}) and user permissions on the destination cluster.
+In this step, you'll start the LDR stream(s) from the destination cluster. You can replicate one or multiple tables in a single LDR job. You cannot replicate system tables in LDR, which means that you must manually apply configurations and cluster settings, such as [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}) and user permissions on the destination cluster.
 
 <a id="modes"></a>_Modes_ determine how LDR replicates the data to the destination cluster. There are two modes:
 
 - `immediate` (default): {% include {{ page.version.version }}/ldr/immediate-description.md %}
 - `validated`: {% include {{ page.version.version }}/ldr/validated-description.md %}
 
-1. From the **destination** cluster, start LDR. Use the fully qualified table name for the source and destination tables:
+### Syntax
 
-    {% include_cached copy-clipboard.html %}
-    ~~~ sql
-    CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.source_table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.destination_table_name};
-    ~~~
+There are two different SQL statements for starting an LDR stream, depending on your requirements:
+
+- {% include_cached new-in.html version="v25.1" %} [`CREATE LOGICALLY REPLICATED`]({% link {{ page.version.version }}/create-logically-replicated.md %}): Creates the new table on the destination cluster automatically, and conducts a fast, offline initial scan. `CREATE LOGICALLY REPLICATED` accepts `unidirectional` or `bidirectional on` as an option in order to create either one of the setups automatically. **The table cannot contain a [user-defined types]({% link {{ page.version.version }}/enum.md %}) or [foregin key]({% link {{ page.version.version }}/foreign-key.md %}) dependencies.** Follow [these steps](#create-logically-replicated) for setup instructions.
+- [`CREATE LOGICAL REPLICATION STREAM`]({% link {{ page.version.version }}/create-logical-replication-stream.md %}): Starts the LDR stream after you've created the matching table on the destination cluster. **If the table contains user-defined types or foreign key dependencies, you must use this syntax.** Allows for manual creation of unidirectional or bidirectional LDR. Follow [these steps](#create-logical-replication-stream) for setup instructions.
+
+Also, for either syntax, note:
```

Comment:
suggest replacing "either" with something non-number-specific like "whichever syntax you use, note"